etc/picongpu/taurus-tud/V100_picongpu.profile.example

```diff
@@ -0,0 +1,80 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ######################################### (edit those lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="ALL"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# Modules #####################################################################
+#
+module purge
+module load modenv/ml
+# similar to foss/2018a, but also includes SpectrumMPI which basically is just a OpenMPI-fork of IBM
+module load gsolf/2018a
+module load GCC/6.4.0-2.28
+module load CMake/3.10.2-GCCcore-6.4.0
+# CUDA is no module in the current enviroment!
+#module load CUDA/9.2.88  # gcc <= 7, intel 15-17
+# OpenMPI is already loaded
+#module load OpenMPI/2.1.2-GCC-6.4.0-2.28
+module load git/2.18.0-GCCcore-6.4.0
+module load zlib/1.2.11-GCCcore-6.4.0
+
+# Self-Build Software #########################################################
+#
+# needs to be compiled by the user
+export PIC_LIBS="/scratch/p_electron/debus/power9/lib"
```

Comment:
This still points to your code

Reply:
Changed to the more generic location `$HOME/lib`.